.vscode/settings.json

```diff
@@ -1,21 +1,15 @@
 {
   "[ruby]": {
     "editor.autoClosingBrackets": "beforeWhitespace",
+    "editor.formatOnType": true,
+    "editor.insertSpaces": true,
+    "files.insertFinalNewline": true,
```

Comment:
Let's remove this particular setting. This contradicts with `trimFinalNewlines` I think.